src/main/java/com/google/growpod/data/User.java

```diff
@@ -0,0 +1,105 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.growpod.data;
+
+/** User data class. */
+public class User {
+
+  public static final int MAX_BIOGRAPHY_LEN = 160;
+
+  /** A unique id. */
+  private String id;
+
+  private String email;
+  /** This is the only name we track. */
+  private String preferredName;
+
+  private String biography;
+  /**
+   * We do not wish to keep exact user locations -- this will store some approximate location data.
+   */
+  private String coarseLocation;
+
+  /**
+   * Constructs a new User.
+   *
+   * @param id A unique ID for the user. This value must be supplied by the user.
+   * @param email The user's email.
+   * @param preferredName The user's preferred name.
+   * @param biography A user-submitted biography truncated to MAX_BIOGRAPHY_LEN.
+   * @param coarseLocation The user's approximate location.
```

Comment:
I would also include how the coarseLocation will be formatted.
For example:
{City}, {State abbreviation}

Reply:
I don't know how this will be formatted yet.

Reply:
+1 What do you intend the course location to be? I personally think it should be the user's zip code so we can match them to gardens in their surrounding area.

Reply:
Yeah, it should be the zip code. I will update all relevant code and documentation.